### ISSUE TITLE
No more work on safari/mac

### ISSUE THREAD

**Issue by electroheadfx** [antfu/icones#86]
### Describe the bug

error with blank page on safari :
TypeError: Object.hasOwn is not a function. (In 'Object.hasOwn(r.prototype,n)', 'Object.hasOwn' is undefined)

### Reproduction

run safari browser with the https://icones.js.org

### System Info

```Shell
OSX 10.14.6 / Safari 14.1.2
but work on opera, chrome and firefox mac
```

### Used Package Manager

n/a

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by antfu** [comment]
/cc @cyberalien I assume it's introduced in `iconify-icon`?

As a note the syntax is supported on my Safari 15.5 and the site works fine for me

**Comment by cyberalien** [comment]
Yes, `Object.hasOwn` requires Safari 15.4. I quickly looked at code that uses it, usage of `hasOwn()` is not necessary, so I'll fix that.

**Comment by cyberalien** [comment]
Fixed in `iconify-icon` 1.0.0-beta.3. 

Fix also includes changes in `package.json` that were discussed on Discord, so Vite should pick correct file now.
